source/_integrations/azure_openai_conversation.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: Azure OpenAI Conversation
+description: Instructions on how to integrate Azure OpenAI as a conversation agent
+ha_category:
+  - Voice
+ha_release: 2024.7
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@zweckj'
+ha_domain: azure_openai_conversation
+ha_integration_type: service
+ha_platforms:
+  - conversation
+---
+
+The Azure OpenAI integration adds a conversation agent powered by [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service) in Home Assistant.
+
+Controlling Home Assistant is done by providing the AI access to the Assist API of Home Assistant. You can control what devices and entities it can access from the {% my voice_assistants title="exposed entities page" %}. The AI is able to provide you information about your devices and control them.
+
+This integration does not integrate with [sentence triggers](/docs/automation/trigger/#sentence-trigger).
+
+This integration requires an Azure OpenAI instance, with an active model deployed.
```

Comment:
By instance, you mean account? Do I need to have login credentials or a token ready or anything like that, before I can add the integration to Home Assistant?